src/scripts/main.js

```diff
@@ -1 +1,63 @@
 'use strict';
+
+const successEl = (message) =>
+  `<div data-qa="notification" class="success">${message}</div>`;
+const errorEl = (message) =>
+  `<div data-qa="notification" class="error">${message}</div>`;
+
+const promiseHandler = (message) => {
+  const body = document.querySelector('body');
+
+  body.insertAdjacentHTML('beforeend', message);
+};
+
+const promise1 = new Promise((resolve, reject) => {
+  const timeoutId = setTimeout(() => {
+    reject(errorEl('First promise was rejected'));
+  }, 3000);
+
+  document.addEventListener('click', () => {
+    clearTimeout(timeoutId);
+    resolve(successEl('First promise was resolved'));
+  });
+});
+
+const promise2 = new Promise((resolve) => {
+  document.addEventListener('click', () => {
+    resolve(successEl('Second promise was resolved'));
+  });
+
+  document.addEventListener('contextmenu', () => {
+    resolve(successEl('Second promise was resolved'));
+  });
+});
```

Comment:
The same issue is present here. You are adding two event listeners but not removing them when the Promise is resolved. Consider removing the event listeners once the Promise is resolved.